workflows/differentialabundance.nf

```diff
@@ -148,6 +149,17 @@ workflow DIFFERENTIALABUNDANCE {
     // Channel for the contrasts file
     ch_contrasts_file = Channel.from([[exp_meta, file(params.contrasts)]])
 
+
+    // Run module to validate models from yml file
+    if ( params.contrasts.endsWith(".yaml") || params.contrasts.endsWith(".yml") ) {
+
+        VALIDATE_MODEL (
+            ch_input,
+            ch_contrasts_file
+        )
+
+    }
```

Comment:
versions file is being generated by the module but not being added to the multiqc versions channel